### PR TITLE
added fileread.js,filesWrite.js,questionlist.txt,toda.md and updated …

### PR DESCRIPTION
**Pull request by pkesavulu**
…fewfiles

### CODE REVIEW

File: public/scripts/homepage.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+// function submitFeedback() {
+//     document.getElementById("nextQuestion").submit();
+// }
+
+function myFunction() {
+    var x = document.getElementById("opt").selectedIndex;
+    var options = document.getElementsByTagName("option")[x].value;
+    var fileName = options.slice(0, -3);
+    // alert(url)
+    location.href = "http://localhost:4000/flash/" + fileName + "/submitFeedback";
```
Comment (mohanmca) [inline public/scripts/homepage.js:16]: how will it work when we deploy to cloud?

Comment (pkesavulu) [reply public/scripts/homepage.js:16]: ok i can change this

Comment (pkesavulu) [reply public/scripts/homepage.js:16]: can you check now everything works perfectly

Comment (pkesavulu) [reply public/scripts/homepage.js:16]: if any changes required please update me will do it

**Review by mohanmca — APPROVED** [review]

**Pull request merged**